README.md

```diff
@@ -30,15 +30,13 @@ jobs:
     runs-on: ubuntu-16.04
     strategy:
       matrix:
-        ruby: [ '2.x', '3.x' ]
+        ruby: [ '2.5.x', '2.6.x' ]
```

Comment:
just to make sure (since I'm not a ruby expert), 3.x doesn't exist, right?  If so, then this makes alot of sense.

Reply:
Exactly, there is no Ruby 3 version yet: https://en.wikipedia.org/wiki/Ruby_(programming_language)#Table_of_versions
And every Ruby software will want to bind to a given minor number, so 2.5.x and 2.6.x is a good example.